tests/run_bf.sh

```diff
@@ -15,4 +16,14 @@ else
 fi
 export PYTHONPATH
 
-exec bft -c "${bf_plugins_dir}"/boardfarm_prplmesh/prplmesh_config.json -n prplmesh_docker -x test_flows
+bft -c "${bf_plugins_dir}"/boardfarm_prplmesh/prplmesh_config.json \
+        -n prplmesh_docker -x test_flows -o "${resultdir}" \
+	|| exit 255
+
+failed_test_count=$(jq '.tests_fail' "${resultdir}"/test_results.json)
+re='^[0-9]+$'
+if ! [[ "$failed_test_count" =~ $re ]]; then
+   echo "Unable to parse failed test count:" "$failed_test_count" \
+   && exit 255
+fi
+exit "$failed_test_count"
```

Comment:
On my ubuntu machine `exit 256` actually returns 0.
Not sure if we are going to fail 256 tests at once, but technically there is integer overflow.

Reply:
I don't we will have that many tests but I'll do the necessary check in bash.

Reply:
I fixed max failed test count to return to 254